detox/e2e/test/smoke_test/login_ldap.e2e.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Ldap, System} from '@support/server_api';
+import {serverUrl} from '@support/test_config';
+import ldapUsers from '@support/fixtures/ldap_users.json';
+
+describe('Smoke Tests', () => {
+    const testOne = ldapUsers['test-1'];
+
+    beforeAll(async () => {
+        // * Verify that the server has license with LDAP feature
+        await System.apiRequireLicenseForFeature('LDAP');
+
+        // # Enable LDAP
+        await System.apiUpdateConfig({LdapSettings: {Enable: true}});
+
+        // * Check that LDAP server can connect and is synchronized with Mattermost server
+        await Ldap.apiRequireLDAPServer();
```

Comment:
Will show an error when not able to connect/sync with LDAP server.
<img width="667" alt="Screen Shot 2020-10-03 at 8 25 19 PM" src="https://user-images.githubusercontent.com/5334504/94991683-2e5e0600-05b7-11eb-9c58-37c02106a104.png">